docs/development/documentation.rst

```diff
@@ -45,3 +45,39 @@ Example usage:
 
     poetry run tidy3d develop replace-in-files -d ./ -j ./docs/versions/test_replace_in_files.json -v 0.18.0 --dry-run True
 
+
+**Command Details**
```

Comment:
So actually I made this for the web api because it changed between 2.5 and 2.6. I then realised that it was backward compatible after I had made this :facepalm: . Could come handy from 2 -> 3 most likely to be honest and in general beneficial to have this since it can update all docs throughout all repos submoduled.

Reply:
I see, thanks!